quickstarts/pom.xml

```diff
@@ -1,230 +1,233 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
The same problem as in SilverWare repository. This file has been reformatted using 2-space indents instead of 3 spaces defined in our code style and so it is hard to see the changes.